features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -513,7 +513,13 @@
 
     <feature name="openhab-binding-netatmo" description="Netatmo Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
-        <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.netatmo/${project.version}</bundle>
+        <bundle dependency="true">wrap:mvn:org.json/json/20131018$Bundle-Name=JSON%20in%20Java&amp;Bundle-SymbolicName=org.json.json&amp;Bundle-Version=20131018</bundle>
+        <bundle dependency="true">wrap:mvn:com.squareup.okhttp/okhttp/2.3.0$Bundle-Name=OkHttp&amp;Bundle-SymbolicName=com.squareup.okhttp&amp;Bundle-Version=2.3.0</bundle>
+        <bundle dependency="true">wrap:mvn:com.squareup.okio/okio/1.3.0$Bundle-Name=Okio&amp;Bundle-SymbolicName=com.squareup.okio&amp;Bundle-Version=1.3.0</bundle>
+        <bundle dependency="true">wrap:mvn:com.squareup.retrofit/retrofit/1.9.0$Bundle-Name=Retrofit&amp;Bundle-SymbolicName=com.squareup.retrofit&amp;Bundle-Version=1.9.0</bundle>
+        <bundle dependency="true">wrap:mvn:org.apache.oltu.oauth2/org.apache.oltu.oauth2.client/1.0.0$Bundle-Name=Apache%20Oltu%20-%20OAuth%202.0%20-%20Client&amp;Bundle-SymbolicName=oorg.apache.oltu.oauth2-client&amp;Bundle-Version=1.0.0</bundle>
+        <bundle dependency="true">wrap:mvn:org.apache.oltu.oauth2/org.apache.oltu.oauth2.common/1.0.0$Bundle-Name=Apache%20Oltu%20-%20OAuth%202.0%20-%20Common&amp;Bundle-SymbolicName=org.apache.oltu.oauth2-common&amp;Bundle-Version=1.0.0</bundle>
+        <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.netatmo/${project.version}</bundle>
```

Comment:
I think both oauth bundles are already OSGI bundles. Wrapping is not needed then. Could you check the other bündle, too? You can download the .jar, rename it to .zip and then check the MANIFEST.MF in META-INF. If it has `Bundle-Manifest-Version: 2` in it, you don't need to wrap it.

Reply:
Seems good, changing features accordingly